inst/templates/mapping_template_ARIADNE.csv

```diff
@@ -1184,11 +1184,13 @@ Tier;Category;Variable;Unit;Definition;piam_variable;piam_unit;piam_factor;Comme
 1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|+|Coal;EJ/yr;1000;;0;;
 1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|+|Gas;EJ/yr;1000;;0;;
 1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|+|Oil;EJ/yr;1000;;0;;
+1;energy (electricity);Secondary Energy|Electricity|Fossil;PJ/yr;net electricity production from coal, gas, conventional and unconventional oil;SE|Electricity|Other Fossil|Other Fossil Adjusted;EJ/yr;1000;;0;;
```

Comment:
as three lines above, SE|Electricity|+|Coal is used and not SE|Electricity|Coal|Other Fossil Adjusted, this line here (1187) can actually be deleted again. 
sorry